Update domain-categories.md (#10126)
cloudflare/cloudflare-docs: content/cloudflare-one/policies/gateway/domain-categories.md

You can block security and content categories by creating DNS or HTTP policies. Once you have configured your policies, you will be able to inspect network activity and the associated categories in your Gateway logs.

> {{<Aside type="note" header="Subdomain Category">}}

Reviewer: ```suggestion
{{<Aside type="note" header="Subdomain category">}}
```

> Subdomains that have not been assigned a category will inherit the category of their parent domain. When a subdomain becomes categorised, it will carry only its own cateogry. Subdomains will not both inherit their parent domain categories and also carry their own.{{</Aside>}}

Reviewer: ```suggestion
Subdomains that have not been assigned a category will inherit the category of their parent domain. When Gateway categorizes a subdomain, the subdomain will carry only its own category. Categorized subdomains will not inherit their parent domain's categories.{{</Aside>}}
```
